src/priorityQueue.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+class Node {
+  constructor(follows, data, priority, proceeds) {
+    this.after = follows
```

Comment:
I would consider renaming the parameter identifiers to be consistent with the private variables - I only recommend this as, when I read through the code, there was a little bit of cognitive load as my brain grappled with the mapping between (for example) `follows` and `after`.

Convention is also to name fields inside of a node `previous`, `next`, and `data`.